ansible/group_vars/psk_orchestration/vars.yml

```diff
@@ -21,4 +21,5 @@ orchestra_apn_key_content_testing: '{{ vault_orchestra_apn_key_content_testing }
 
 orchestra_database_url_testing: 'postgres://{{ orchestra_database_username }}:{{ orchestra_database_password_testing }}@db-1.proteus.test.ooni.io:5432/{{ orchestra_database_name }}?sslmode=require'
 orchestra_database_url: 'postgres://{{ orchestra_database_username }}:{{ orchestra_database_password }}@db-1.proteus.ooni.io:5432/{{ orchestra_database_name }}?sslmode=require'
+orchestra_psiphon_config_file_content: '{{ vault_orchestra_psiphon_config_file_content }}'
```

Comment:
This group `psk_orchestration` actually is not used for probe-services hosts: https://github.com/ooni/sysadmin/blob/master/ansible/inventory#L79.

I suggest you use the group `psk_orchestration_prod` and `psk_orchestration_test`

Reply:
fefba87